shell/src/main/java/alluxio/cli/fsadmin/command/report/CapacityCommand.java

```diff
@@ -9,7 +9,7 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.cli.fsadmin.report;
+package alluxio.cli.fsadmin.command.report;
```

Comment:
could we put all the commands under the `command` package instead of having a different package per command? Then it would be consistent with how the filesystem shell works

Reply:
fsadmin doctor and fsadmin report both have ConfigurationCommand under them, so we can DoctorConfigurationCommand, ReportConfigurationCommand to separate those subcommands?

Reply:
what do you think of `alluxio.cli.fsadmin.report.ConfigurationCommand`, `alluxio.cli.fsadmin.doctor.ConfigurationCommand`?

Reply:
Changed, thanks